doc/Command-Reference.md

```diff
@@ -2315,6 +2317,33 @@ Optional argument "-p" specify a period (in seconds) with which to gather counte
    Ethernet24        U   33,543,533,441   36.59 MB/s      0.71%         0     1,613         0   43,066,076,370   49.92 MB/s      0.97%         0         0         0
   ```
 
+The "errors" subcommand is used to display the interface errors. 
+
+- Example:
+  ```
+  admin@str-s6000-acs-11:~$ show interface counters errors
+      IFACE    STATE    RX_ERR    RX_DRP    RX_OVR    TX_ERR    TX_DRP    TX_OVR
+  -----------  -------  --------  --------  --------  --------  --------  --------
+    Ethernet0        U         0         4         0         0         0         0
+    Ethernet4        U         0         0         0         0         0         0
+    Ethernet8        U         0         1         0         0         0         0
+   Ethernet12        U         0         0         0         0         0         0
+```
+
+The "rate" subcommand is used to disply only the interface rate. 
```

Comment:
Pluralize "rates" here, also.

Reply:
That was already updated.

Reply:
Weird. It wasn't reflected when I last viewed the PR. :/